### ISSUE TITLE
Bug: Dropdown TOCs not working since jQuery upgrade

### ISSUE THREAD

**Issue by daforster** [ryan-roemer/sphinx-bootstrap-theme#227]
The global and local dropdown TOCs do not work properly. You can check this even on your [Sphinx Sample page](https://sphinx-themes.org/sample-sites/sphinx-bootstrap-theme). 

The reason seems to be -- as far as I found out -- that in `_static/js/jquery-fix.js` the variable `jQuery` is unset after `.noConflict(true)` returns.

I am not really familiar with jQuery but it might be a problem with its upgrade. On [Bootstrap Theme Demo](https://sphinx-bootstrap-theme.readthedocs.io) under the older jQuery version it still works fine.

**Comment by johentsch** [comment]
Yes, same here: https://dcmlab.github.io/standards Any chance this breaking bug is addressed soon?

**Comment by tasusu** [comment]
Same here. Simply removing `_static/js/jquery-fix.js` works for me.
